### PR TITLE
Fixed database connection

### PR DESCRIPTION
**Pull request by Bijelic03**

### CODE REVIEW

File: src/main/java/com/internship/socialnetwork/model/User.java [hunk 1]
```diff
@@ -1,32 +1,32 @@
 package com.internship.socialnetwork.model;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
-import jakarta.persistence.ManyToMany;
-import jakarta.persistence.OneToMany;
-import lombok.*;
+import jakarta.persistence.*;
```
Comment (markonikolic-nqode) [inline src/main/java/com/internship/socialnetwork/model/User.java]: Pls avoid star imports

**Review by markonikolic-nqode — CHANGES_REQUESTED** [review]

File: src/main/java/com/internship/socialnetwork/model/User.java [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+import lombok.NoArgsConstructor;
 
+import java.io.Serializable;
 import java.util.List;
 
 @Data
 @NoArgsConstructor
 @Entity
-public class User {
+public class User implements Serializable {
```
Comment (markonikolic-nqode) [inline src/main/java/com/internship/socialnetwork/model/User.java:19]: Still not working without Serializable?

Comment (Bijelic03) [reply src/main/java/com/internship/socialnetwork/model/User.java:19]: I tried to make it without, but it didn't work.

**Review by markonikolic-nqode — APPROVED** [review]

File: src/main/java/com/internship/socialnetwork/model/User.java (hunk 2 above)
Comment (markonikolic-nqode) [reply src/main/java/com/internship/socialnetwork/model/User.java:19]: Okay, let's move on like this for now, but it's a workaround, not a real solution, just FYI.

**Pull request merged**